### PR TITLE
#822 gRPC layer removal: update tanka deployment; update doc and some diagrams

### PR DESCRIPTION
**Pull request by mickmis**
This updates the Tanka deployment and associated documentation.
I could test this works with a local k3s-based deployment. This leaves out some things compared to doing a cloud deployment and/or a deployment that joins a pool, so a thorough review would be appreciated (ping @marcadamsge).

### CODE REVIEW

**Review by BenjaminPelletier — APPROVED** [review]
This looks good to me, but I'm a very weak reviewer for technical deployment things (the *.*sonnet files) -- if @marcadamsge could take a look too (probably focusing on *.*sonnet files if time is limited), I think that would be super helpful

File: assets/pool_architecture.gv [hunk 1]
```diff
@@ -39,54 +33,54 @@ digraph G {
...
+    CoreService1b -> CRDB1b;
     }
     CRDB1a -> PrivateKey1 [dir=back,style=dotted];
     CRDB1b -> PrivateKey1 [dir=back,style=dotted];
-    HTTPGateway1a -> CoreService1b -> CRDB1a;
-    HTTPGateway1b -> CoreService1a -> CRDB1b;
-    LoadBalancer1 -> HTTPGateway1a
-    LoadBalancer1 -> HTTPGateway1b
+    CoreService1a -> CoreService1b -> CRDB1a;
+    CoreService1b -> CoreService1a -> CRDB1b;
```
[the remark this answers is not in the document]
Comment (mickmis) [reply assets/pool_architecture.gv:45]: Indeed

File: assets/pool_architecture.gv [hunk 2]
```diff
@@ -39,54 +33,54 @@ digraph G {
     label="USS1's DSS instance"
     subgraph cluster_1 {
     label="Stateful set a";
-    HTTPGateway1a -> CoreService1a -> CRDB1a;
+    CoreService1a -> CRDB1a;
```
[the remark this answers is not in the document]
Comment (mickmis) [reply assets/pool_architecture.gv:36]: Indeed, I did not notice the transparent buffer.

File: build/deploy/core-service.libsonnet [hunk 3]
```diff
@@ -1,11 +1,56 @@
...
+  PresharedCertIngress(metadata, certName): ingress(metadata) {
+    metadata+: {
+      annotations+: {
+        'ingress.gcp.kubernetes.io/pre-shared-cert': certName,
+      },
+    },
+  },
+
   all(metadata): {
     service: base.Service(metadata, 'core-service') {
```
Comment (marcadamsge) [inline build/deploy/core-service.libsonnet:50]: I think you're missing `ingress: $.ManagedCertIngress(metadata)` here, there's not Ingress in the rendered templates

**Review by marcadamsge — COMMENTED** [review]
It could be nice to update the [CHANGELOG](https://github.com/interuss/dss/blob/master/build/CHANGELOG.md) as well. Except for the Ingress, everything looks good to me 👍

File: build/deploy/core-service.libsonnet (hunk 3 above)
Comment (mickmis) [reply build/deploy/core-service.libsonnet:50]: Very good catch, thanks

**Comment by mickmis** [comment]
> It could be nice to update the [CHANGELOG](https://github.com/interuss/dss/blob/master/build/CHANGELOG.md) as well. Except for the Ingress, everything looks good to me +1

Done :+1:

**Review by marcadamsge — APPROVED** [review]

**Pull request merged**